ui/src/screens/SearchScreen/SearchScreen.jsx

```diff
@@ -129,10 +131,11 @@ class SearchScreen extends React.Component {
         icon: 'person'
       }
     ];
-    this.state = {facets: facets};
+    this.state = {facets: facets, toggle: false};
```

Comment:
can we rename toggle -> hideFacets or something? Toggle is too generic as a name.